critiquebrainz/frontend/templates/user/base.html

```diff
@@ -30,3 +30,8 @@ <h2 class="pull-left"><img class="avatar" src="{{ user.avatar }}&s=32" /> {{ use
     {% block profile_content %}{% endblock %}
   </div>
 {% endblock %}
+
+{% block scripts %}
+  {{ super() }}
+  <script src="{{ get_static_path('rating.js') }}"></script>
```

Comment:
This script is not used in this file and user/info.html inherits from this template and does not use this script either. So I suggest to move this block to reviews.html which is the file actually needing the script.

Reply:
Sure, I'll do that.